core/src/main/java/org/ironjacamar/core/api/deploymentrepository/ResourceAdapter.java

```diff
@@ -64,6 +66,37 @@
    public Recovery getRecovery();
 
    /**
+    * Does the resource adapter support the message listener
+    * @param ml The class name of the message listener
+    * @return True if there is support, otherwise false
+    */
+   public boolean supportsMessageListener(String ml);
+
```

Comment:
what about calling it like isSupportingMessageListener(String ml)? Even if the "s" should make it clear, current method name could be easily misread at a first look.